chia/_tests/core/mempool/test_singleton_fast_forward.py

```diff
@@ -565,3 +566,64 @@ async def test_singleton_fast_forward_same_block() -> None:
         assert unspent_lineage_info.parent_id == latest_singleton.parent_coin_info
         # The one before it should have the second last random amount
         assert unspent_lineage_info.parent_amount == random_amounts[-2]
+
+
+@pytest.mark.anyio
+async def test_singleton_fast_forward_solo() -> None:
+    """
+    We don't allow a spend bundle with *only* fast forward spends, since those
+    are difficult to evict from the mempool. They would always be valid as long as
+    the singleton exists.
+    """
+    START_AMOUNT = uint64(1337)
+    # We're decrementing the next iteration's amount for testing purposes
+    SINGLETON_AMOUNT = uint64(1335)
+    # We're incrementing the next iteration's amount for testing purposes
+    SINGLETON_CHILD_AMOUNT = uint64(1339)
```

Comment:
I understand this was modeled after an existing test, but for this one we don't need to alter the singleton amount across iterations, so we can simplify things here.